osf/management/commands/data_storage_usage.py

```diff
@@ -31,7 +32,7 @@
     'target_object',
     'target_content_type_id',
     'deleted_on',
-    'size',
+    'size (b)',
```

Comment:
Could you verify with Courtney that changing the header won't break her scripts?

Reply:
Good call - changing the header back.